src/pykeen/datasets/base.py

```diff
@@ -122,6 +124,22 @@ def get_normalized_name(cls) -> str:
         """Get the normalized name of the dataset."""
         return normalize_string(cls.__name__)
 
+    def similarity(self, other: Dataset) -> float:
```

Comment:
maybe it would be more extensible to have a method outside of the dataset class itself, which takes two datasets as input and outputs a scalar similarity value?

Right now, it just uses a very specific definition of dataset similarity (jaccard similarity on the sets of triples if I see it correctly).

Reply:
Well this does just defer to that outside measure. But I guess you’re right - this is pretty limiting.

Reply:
Also I implemented this really funny way doing 1 - xor/Union rather than directly intersection/Union 😂

Reply:
The reason was that this gives the intuition of how many swaps you need to make to morph one split into another. Later I will reimplement this to also take into account splits of more than length 2, then the correspondence to tanimoto will go away (so I will leave it as it is)

Reply:
Addressed in d73a71c3